index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ] [Optional] As mentioned in the project requirements, I request that you include a `<header>` section in the html file
<img width="524" alt="req" src="https://user-images.githubusercontent.com/4247740/143236455-9922d4c7-a910-4fc9-94a6-60ba494a157f.PNG">